### PR TITLE
adding greenhill uto-aztecan concept list

### PR DESCRIPTION
**Pull request by MuffinLinwist**
# Pull request checklist

- [x] add new concept list
- [x] add new metadata
- [ ] add new Concepticon concept sets
  * [ ] checked whether the new concept(s) can be applied to existing lists with
    `concepticon notlinked --gloss "NEW_GLOSS"`
- [ ] add new Concepticon concept relations
- [ ] refine existing Concepticon concept set mappings
- [ ] refine Concepticon glosses
- [ ] refine Concepticon concept relations
- [ ] refine Concepticon concept definitions
- [ ] retire data

## Additional information

...

Here is the concept list from the lexibank repo of uto-aztecan. I have three small suggestions on the mappings. Since they were already worked through, I'm just putting them as comments in this PR. @FredericBlum can I ask you to moderate this PR?

### CODE REVIEW

File: concepticondata/conceptlists/Greenhill-2023-121.tsv [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+Greenhill-2023-121-41	41	feather	1201	FEATHER
+Greenhill-2023-121-42	42	hair	1040	HAIR
+Greenhill-2023-121-43	43	head	1256	HEAD
+Greenhill-2023-121-44	44	ear	1247	EAR
+Greenhill-2023-121-45	45	eye	1248	EYE
+Greenhill-2023-121-46	46	nose	1221	NOSE
+Greenhill-2023-121-47	47	mouth	674	MOUTH
+Greenhill-2023-121-48	48	tooth	1380	TOOTH
+Greenhill-2023-121-49	49	tongue	1205	TONGUE
+Greenhill-2023-121-50	50	claw, nail	72	CLAW
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Greenhill-2023-121.tsv:51]: CLAW OR NAIL?

File: concepticondata/conceptlists/Greenhill-2023-121.tsv [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+Greenhill-2023-121-59	59	breast	1402	BREAST
+Greenhill-2023-121-60	60	heart	1223	HEART
+Greenhill-2023-121-61	61	liver	1224	LIVER
+Greenhill-2023-121-62	62	drink	1401	DRINK
+Greenhill-2023-121-63	63	eat	1336	EAT
+Greenhill-2023-121-64	64	bite	1403	BITE
+Greenhill-2023-121-65	65	vomit	1278	VOMIT
+Greenhill-2023-121-66	66	see	1409	SEE
+Greenhill-2023-121-67	67	hear	1408	HEAR
+Greenhill-2023-121-68	68	know	1410	KNOW (SOMETHING)
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Greenhill-2023-121.tsv:69]: KNOW?

File: concepticondata/conceptlists/Greenhill-2023-121.tsv [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+Greenhill-2023-121-81	81	sing	1261	SING
+Greenhill-2023-121-82	82	sun	1343	SUN
+Greenhill-2023-121-83	83	moon	1313	MOON
+Greenhill-2023-121-84	84	star	1430	STAR
+Greenhill-2023-121-85	85	sky	1732	SKY
+Greenhill-2023-121-86	86	water	948	WATER
+Greenhill-2023-121-87	87	wind	960	WIND
+Greenhill-2023-121-88	88	rain	658	RAIN (PRECIPITATION)
+Greenhill-2023-121-89	89	snow	784	SNOW
+Greenhill-2023-121-90	90	stone, rock	857	STONE
```
Comment (MuffinLinwist) [inline concepticondata/conceptlists/Greenhill-2023-121.tsv:91]: STONE OR ROCK?

File: concepticondata/conceptlists/Greenhill-2023-121.tsv (hunk 2 above)
Comment (LinguList) [reply concepticondata/conceptlists/Greenhill-2023-121.tsv:69]: Here, one could also check the individual data points, to see if it is rather "know (something)". The "prototypcial" KNOW is always "KNOW (SOMETHING)", however.

File: concepticondata/conceptlists/Greenhill-2023-121.tsv (hunk 3 above)
Comment (LinguList) [reply concepticondata/conceptlists/Greenhill-2023-121.tsv:91]: Yes, stone or rock is good.

**Comment by LinguList** [comment]
I added few inline comments.

**Comment by MuffinLinwist** [comment]
Thanks, @LinguList. I am applying them right now.

File: concepticondata/conceptlists/Greenhill-2023-121.tsv (hunk 2 above)
Comment (MuffinLinwist) [reply concepticondata/conceptlists/Greenhill-2023-121.tsv:69]: I checked and it seems that the best mapping for this is `KNOW (SOMETHING)` indeed. I'll, therefore, leave it as it is.

**Review by LinguList — APPROVED** [review]

**Comment by AnnikaTjuka** [comment]
@MuffinLinwist What is the status of this PR? Can it be merged?

**Comment by FredericBlum** [comment]
@MuffinLinwist Reminder about this PR to check if merge-ready.

**Comment by SimonGreenhill** [comment]
thanks @MuffinLinwist !

**Pull request merged**